edc-extensions/bpn-validation/bpn-validation-core/src/main/java/org/eclipse/tractusx/edc/validation/businesspartner/functions/BusinessPartnerGroupFunction.java

```diff
@@ -115,13 +114,13 @@ public boolean evaluate(Operator operator, Object rightOperand, Permission permi
         }
 
         // right-operand is anything other than String or Collection
-        var rightOperand1 = parseRightOperand(rightOperand, context);
-        if (rightOperand1 == null) {
+        var parsedRightOperand = parseRightOperand(rightOperand, context);
```

Comment:
this `parseRightOperand` could already return a `Set<String>`

Reply:
you're right, missed that, changed [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1673/commits/8541be9d7cfe82892161b63ab86477d36e10525a).